astropy/tests/helper.py

```diff
@@ -220,8 +220,6 @@ def treat_deprecations_as_exceptions():
     # this iteration thus we copy the original state to a list to iterate
     # on. See https://github.com/astropy/astropy/pull/5513.
     for module in list(sys.modules.values()):
-        # We don't want to deal with six.MovedModules, only "real"
-        # modules. FIXME: we no more use six, this should be useless ?
```

Comment:
Well, I think it means the `if` statement is actually useless now? Isn't that what @mhvk mean in the other PR?

Reply:
If so, this deletion should be part of #12038

Reply:
I think what makes the most sense is to remove one clause of the if statement in this PR.

Reply:
> If so, this deletion should be part of #12038

Oops! Please take a look at what I just did in this PR. If you don't agree, then I'll close this one and move it into #12038.

Reply:
I still think this should be part of #12038 . Otherwise, this will cause conflict in your other PR and we have to deal with logistics of 2 PRs for a few lines of code. If you strongly disagree, we can settle this PR first and I'll mark your other PR as being blocked by this one.

Reply:
Done.